frontend/src/tests/session-tests.js

```diff
@@ -27,6 +27,7 @@ export function sessionsTests(api) {
 
     beforeAll(async () => {
         await apiPOST("/debug/restore_snapshot");
+        session = databaseSeeder.seededData.session;
```

Comment:
Out of curiosity, did it still not work when you assign it before the `beforeAll` and used it inside of each test?